lib/core-integration/src/Test/Integration/Scenario/API/Shared/Transactions.hs

```diff
@@ -195,6 +213,22 @@ spec = describe "SHARED_TRANSACTIONS" $ do
             , expectField (#fee . #getQuantity) (`shouldSatisfy` (>0))
             ]
 
+        let txCbor = getFromResponse #transaction rTx2
```

Comment:
I think this integration test now does more than just create a transaction. Could you make this apparent in the test description? 🤔 I.e.

```
    it "SHARED_TRANSACTIONS_CREATE_01 - Can create and decode tx for an active shared wallet" $ \ctx -> runResourceT $ do
```

in line 165 (to which I cannot attach a comment).

Reply:
well, we create tx and check whether resultant serialized binary is as expected. We use the same flow in shelley new tx workflow